docs/en/observability/create-alerts.asciidoc

```diff
@@ -3,39 +3,66 @@
 
 [IMPORTANT]
 ====
-Ensure Alerting is already set up in {kib}. For details, see
+Make sure alerting is already set up in {kib}. For details, see
 {kibana-ref}/alerting-setup.html[Setup and prerequisites].
 ====
 
-Alerting enables you to detect complex *conditions* defined by a rule
+Alerting enables you to detect complex conditions defined by a *rule*
 within the Logs, Metrics, Uptime, and APM apps. When a condition is met, the rule
 tracks it as an *alert* and responds by triggering one or more *actions*.
 
-Listed on the <<view-observability-alerts,Alerts>> page are all of your
-observability alerts that have met a condition defined by a rule you created
-using the Logs, Metrics, Uptime, or APM apps.
+Alerts related to {observability} apps including Logs, Metrics, Uptime, and APM can be managed in the {observability} UI.
+You can also manage {observability} app rules alongside rules for other apps from the {kibana-ref}/create-and-manage-rules.html[{kib} Management UI].
 
-IMPORTANT: Not all of the predefined rules in Stack Management will generate and 
-list an alert on the Alerts page. Only alerts generated by rules relating to Logs, Metrics, Uptime, and APM
-can be viewed on the Alerts page.
+[discrete]
+[[create-alerts-rules]]
+== Create rules
+
+The first step when setting up alerts is to create a rule.
+To create and manage rules related to {observability} apps,
+go to the {observability} *Alerts* page and click *Manage Rules* to navigate to the {observability} Rules page.
+
+[role="screenshot"]
+image::images/create-alerts-manage-rules.png[Elastic {observability} Rules page]
 
-Alerting can be centrally managed from the
-{kibana-ref}/create-and-manage-rules.html[{kib} Management UI] and provides a
-set of built-in {kibana-ref}/rule-types.html[rule types] and
-{kibana-ref}/action-types.html[connectors] for you to use.
+From the {observability} Rules page, you can manage rules for {observability} apps, including:
+
+* Creating a new rule
+* Editing or deleting existing rules
+* Updating the status of existing rules
```

Comment:
@colleenmcginnis I would like to mention a detail regarding rule status and you decide if we need to mention it or not. At 8.2 Response Ops team introduced the new `snoozed` status https://github.com/elastic/kibana/issues/124888. Because the API was not ready for us to use in time, we agreed to use only the `Snooze indefinitely` option in observability rules page. If users want to snooze a rule for a specific time period, they should use the Stack Management > Rules and Connectors page.